### ISSUE TITLE
Members Only: Possible inaccurate statement in intro paragraph of lesson

### ISSUE THREAD

**Issue by tgorsk15** [TheOdinProject/curriculum#28675]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### Describe your suggestion

The sentence states that the learner will be able to practice their newly learned authentication skills that they learned through the last "project". The last project, Inventory-App, did not use any authentication yet... so I am assuming "project" should be replaced with "lesson", as the Authentication lesson comes right before this page
![image](https://github.com/user-attachments/assets/9e2bdf64-2745-4f5f-a954-e7c4b8898171)

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/node-path-nodejs-members-only

### (Optional) Discord Name

tgorsk15

**Comment by MaoShizhong** [comment]
Yep, "lesson" sounds right. While you're at it, could you end the sentence there and start the next bit as a new sentence? Doesn't really make sense with the comma.

**Comment by MaoShizhong** [comment]
@tgorsk15 Just a courtesy check in if this is still something you wish to work on?

**Comment by tgorsk15** [comment]
Hey @MaoShizhong .  I do want to still do this and get practice when it comes to contributing. I've just been very focused/busy on getting the back half of my projects done in the Node curriculum so I can start exploring the market, while trying to balance my current job , etc..., so I have not made this a priority which is my bad.  

 If this needs to get fixed asap feel free to give someone else. However, I will commit some time over the next week to do this if it is not a rush.  I guess I'm also slightly intimated by it since I've never worked on an open issue before, so I figured I would need to commit some quality time learning it, even though it's just a small typo.   Hope you're having a great weekend!

**Comment by MaoShizhong** [comment]
No worries, it isn't a rush. Just checking in just in case something was forgotten but all good if not!
And you'll likely find when you do get round to it, it's not that much different from what you're used to. A few extra things but much of the fork/clone/branch/push stuff is stuff you've done before.

Do of course ask should you run into any issues.

**Comment by tgorsk15** [comment]
Ok sounds good, I should be able to carve out some time for it in the next week! 
Cool, sounds like it won't be too difficult to pick up, and yes I'll be sure to ask if needed

**Comment by dragon-slayer27** [comment]
@KevinMulhern Hello, it is my first time (trying to) contribute to this repo and I would love to fix this issue could I be assigned?

**Comment by mao-sz** [comment]
Go for it @dragon-slayer27

**Comment by dragon-slayer27** [comment]
I am sorry i was pretty busy and this issue slipped my mind. I have submitted a pr for this issue (#30313)
